guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -46,6 +46,18 @@ endif::[]
 . Click *Submit*.
 This displays the *Job Overview* page, and when the job completes, also displays the status of the job.
 
+You can also set up recurring jobs from the new Host page.
+
+.Procedure
```

Comment:
Can you please move this procedure to a new file? We generally have one procedure per file.